src/Form/MandateType.php

```diff
@@ -41,22 +37,22 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     return 'elected_representative.mandate.larem_support.'.\mb_strtolower($key);
                 },
             ])
-            ->add('beginAt', 'sonata_type_date_picker', [
-                'label' => 'Date de début de mandat',
+            ->add('onGoing', CheckboxType::class, [
+                'label' => false,
+                'required' => false,
             ])
+            ->add('beginAt', 'sonata_type_date_picker')
             ->add('finishAt', 'sonata_type_date_picker', [
                 'required' => false,
-                'label' => 'Date de fin de mandat',
-            ])
-            ->add('geographicalArea', TextType::class, [
-                'label' => 'Périmètre géographique',
             ])
+            ->add('geographicalArea', TextType::class)
         ;
 
         $builder->addEventListener(FormEvents::PRE_SUBMIT, function (FormEvent $event) {
             $data = $event->getData();
 
-            if (!isset($data['isElected']) || false === $data['isElected']) {
+            if (!isset($data['isElected']) || false === $data['isElected']
```

Comment:
What about removing this listener entirely as you have added 2 `@Assert\Expression` on `finishAt` field and JS code for disable/enable the same field?